health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppConfigurationParentLayer.js

```
@@ -260,30 +260,36 @@ const AppConfigurationParentRedesign = ({ formData = null, isNextTabAvailable, i
+                      }
+                      return {
+                        ...j,
+                        active: false,
+                      };
+                    });
+                  });
+                  setCurrentStep(1);
+                }}
+              />
+            </div>
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)